### PR TITLE
JavaScript Capstone Project v 1.0

### PR DESCRIPTION
**Pull request by YKerroum**
- [x] Use JavaScript to manipulate DOM elements.

- [x] Use JavaScript events.

- [x] Create and access data from API.

- [x] Use ECMAScript syntax.

- [x] Use ECMAScript modules to write modular JavaScript.

- [x] Use callbacks and promises.

- [x] Use webpack to bundle JavaScript.

- [x] Use npm as a package manager.

- [x] Use JSON.

- [x] Make JavaScript code asynchronous.

- [x] Write unit tests using Jest.

- [x] Use gitflow.

- [x] Perform a code review for each other.

### CODE REVIEW

File: src/modules/items.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const itemCounter = () => {
+  const list = document.getElementById('movies');
+  const movieCount = document.getElementById('movieNav');
+  movieCount.innerHTML = `Movies (${list.children.length})`;
+};
```
Comment (OyePriscilla) [inline src/modules/items.js:5]: - [ ] Kindly implement the `itemCounter` for the mobile version as well, I could see that you did it only for the desktop version. 👍

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+.popup p {
+  font-size: 13px;
+}
+
+.popComment {
+  margin-bottom: 10px;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (OyePriscilla) [inline src/index.css:209]: - [ ] Please I noticed that on your mobile version the for the comment pop up; the comment button is not showing. I guess you need to adjust the styling of the parent container a bit, perhaps you can add a vertical scroll bar to comment pop up element 😄.

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+  flex-direction: column;
+  text-align: start;
+  width: 100%;
+  height: 100vh;
+  left: -100%;
+  top: 0;
+  transition: 0.3s;
+  backdrop-filter: blur(5px);
+  background-color: rgba(0, 0, 0, 0.774);
+}
```
Comment (OyePriscilla) [inline src/index.css:229]: - [ ] Please I noticed that on your mobile version the for the comment pop up window; the comment button is not showing. I guess you need to adjust the styling of the parent container a bit, perhaps you can add a vertical scroll bar to comment pop up window😄. 

> overflow-y: scroll;

- [ ] Moreso, kindly ensure to design the desktop version for pop up window according to the given wireframe; doing this gives hiring manager good confidence that we can actually deliver whatsoever design template given. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @YKerroum and @sebalp100 😃 , 

Good team job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Descriptive pull request message and title :heavy_check_mark: 
Linter error checks validated :heavy_check_mark: 
Commit messages are meaningful. :heavy_check_mark: 
GitHub flow observed ✔️ 
All tests passed✔️

## Suggested changes

- [ ] Please you need to follow the new [Readme template](https://github.com/microverseinc/readme-template)
- [ ] You need to make a video presentation presenting all features of your project according to the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md).
- [ ] Check the comments under the review for more required changes.

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi team, 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


 ## Required Changes ♻️

- [ ] Hey team, you did a great job on implementing the changes requested by the previous reviewer 👏, but you forgot one thing she requested you: `Recording the video presentaiton`. Please would you mind doing that?

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sebalp100** [comment]
> # Changes Required ♻️
> ## Hi team,
> AlmostThereReachGIF [ AlmostThereReachGIF ](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif) [ ](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif)
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## (Highlights) Good Points: 👍
> * You documented your work professionally. ✔
> * GitHub flow is used correctly. ✔
> * All linters checks are ok. ✔
> 
> ## Required Changes ♻️
> * [ ]  Hey team, you did a great job on implementing the changes requested by the previous reviewer 👏, but you forgot one thing she requested you: `Recording the video presentaiton`. Please would you mind doing that?
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @ridwanediallo , in the requirements for this project (https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/buisness_requirements.md) the only mention of a video recording is for the final assessment. We need to record a 30 min video. I think the video presentation mentioned by the previous reviewer is from an old version. I would appreciate some clarification. Thank you for your time.

**Comment by ridwanediallo** [comment]
Hello @YKerroum, you are right. Please ignore this comment and submit for another review.

**Pull request merged**